src/client/services/board/persistence/persistence.js

```diff
@@ -65,6 +65,50 @@ var PersistenceService = SparkBase.extend({
       }.bind(this));
   },
 
+
+  /**
+   * Adds a snapshot image of the board
+   *
+   * @param {Conversation} conversation - the current conversation that the board belongs
+   * @param {Board~Channel} channel
+   * @param {File} image
+   * @returns {Promise<Board~Channel>}
+   */
+  addSnapshotImage: function addSnapshotImage(conversation, channel, image) {
+    var imageScr;
+    return this.spark.board._uploadImage(conversation, image)
+      .then(function encryptScr(scr) {
+        imageScr = scr;
+        return this.spark.encryption.encryptScr(imageScr, conversation.defaultActivityEncryptionKeyUrl);
+      }.bind(this))
+      .then(function attachEncryptedScr(encryptedScr) {
+        imageScr.encryptedScr = encryptedScr;
+        return encryptedScr;
+      }.bind(this))
+      .then(function addSnapshotToChannel() {
+        var imageBody = {
+          image: {
+            url: imageScr.loc,
+            height: image.height || 900,
+            width: image.width || 1600,
+            mimeType: image.type || 'image/png',
+            scr: imageScr.encryptedScr,
+            encryptionKeyUrl: conversation.defaultActivityEncryptionKeyUrl,
+            fileSize: image.size
+          }
+        };
+        return this.spark.request({
+          method: 'PATCH',
```

Comment:
Why is this a patch? I'm pretty sure POST is the correct verb if I'm understanding the semantics of this action.